cypress/support/dataTestids.js

```diff
@@ -0,0 +1,160 @@
+export const dataTestId = {

```

Comment:
Bare et forslag, så man ikke begynner å oppdatere verdier her.
```suggestion
// This file is readonly. The contents of this file is copied from https://github.com/BIBSYSDEV/NVA-Frontend/blob/main/src/utils/dataTestIds.ts

export const dataTestId = {

```